Password optional sign in javascript node guide (#3437)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/index.md

---
title: Sign up for new account with email only
---

<div class="oie-embedded-sdk">

<ApiLifecycle access="ie" /><br>

> Learn how to use the Embedded SDK to integrate password optional sign up in your app.

Reviewer: ```suggestion
Learn how to use the Embedded SDK to integrate password-optional sign-up flows in your app.
```

Author: Issue fixed in PR: https://github.com/okta/okta-developer-docs/pull/3435

**Learning outcomes**

> * Configure your Okta org to allow your users the option to sign up without a password

Reviewer: ```suggestion
* Configure your Okta org to allow your users the option to sign up without a password.
```

> * Enable your app to support password optional sign ups

Reviewer: ```suggestion
* Enable your app to support password-optional sign-up flows.
```

Author: Issue fixed in PR: https://github.com/okta/okta-developer-docs/pull/3435

**What you need**

<StackSnippet snippet="whatyouneed" />
</br>

**Sample code**

<StackSnippet snippet="samplecode" />

---

## Overview

## Update configurations

> Before you can start integrating password-optional sign-ups in your app, <StackSnippet snippet="setupoktaorg" inline/>. Additionally, you must enable new user sign-ups with the following steps:

Reviewer: ```suggestion
Before you can start integrating password-optional sign-up flows in your app, <StackSnippet snippet="setupoktaorg" inline/>. Additionally, you must enable new user sign-up flows with the following steps:
```

Author: Issue fixed in PR: https://github.com/okta/okta-developer-docs/pull/3435

1. In the Admin Console, go to **Security** > **Profile Enrollment**.
1. Locate the **Default Policy** and select the pencil icon.
1. Under **Enrollment Settings**, select **Actions > Edit**.
1. Under **For new users**, select **Allowed** for **Sign-up**.
1. Click **Save**.

## Integrate

### Summary of steps

> The following summarizes the steps involved in the password-optional sign-up.

Reviewer: ```suggestion
The following summarizes the steps involved in the password-optional sign-up flow.
```
